tests/end2end/features/test_misc_bdd.py

```diff
@@ -29,6 +29,7 @@
 from qutebrowser.utils import docutils
 from qutebrowser.browser import pdfjs
 
+from end2end.features.test_scroll_bdd import check_scrolled, check_not_scrolled
```

Comment:
Hmm, I'd really expect this to work, and yet it doesn't. I'll investigate later, though it might get Monday until I get the time.